CHANGELOG.md

```diff
@@ -37,6 +37,13 @@ In Strimzi 0.12.0, the `v1alpha1` versions of the following resources have been
 In the next release, the `v1alpha1` versions of these resources will be removed. 
 Please follow the guide for upgrading the resources: https://strimzi.io/docs/operators/latest/full/deploying.html#assembly-upgrade-resources-str.
 
+#### Removal deprecated cadvisor metric labels
+
+The `pod_name` and `container_name` labels provided on the cadvisor metrics are now just `pod` and `container` starting from Kubernetes 1.16.
+We removed the old ones from the Prometheus scraping configuration/alerts and on the Kafka and ZooKeeper dashboard as well.
+It means that the charts related to memory and CPU usage are not going to work on Kuvbernetes version previous 1.14.
+For more information see this Kubernetes PR https://github.com/kubernetes/kubernetes/pull/80376
```

Comment:
It would be better to link directly to _this_ PR, that way they can immediately see what got changed.

Reply:
makes more sense ;-)